src/ast/variables/LocalVariable.ts

```diff
@@ -33,17 +37,32 @@ export default class LocalVariable extends Variable {
 		super(name);
 		this.declarations = declarator ? [declarator] : [];
 		this.init = init;
+		this.initialisers = [init];
```

Comment:
I love how this project is committed to being rebelliously European :)

I hate to be difficult,  but surely we shouldn't need to track all initializers here, since initializer precedence can always apply (https://github.com/rollup/rollup/issues/2278)? Or do we specifically need to apply `reassignPath(UNKNOWN_PATH)` to each initializer? Can you perhaps help clarify why tracking this information on the variable is not enough?

Alternatively could `this.init` be replaced to always reference `this.initialisers[0]`? I just hate to see the redundancy.

Reply:
Without TDZ/flow analysis, we do not know which definition applies where. Thus the approach is to make the variable "unknown". As we can no longer track initializer references in such a situation, all initializers need the `reassignPath(UNKNOWN_PATH)`.
I could however simplify this to get rid of the redundancy.